main.yml

```diff
@@ -696,14 +699,22 @@ streams:
       - id: kafkas
         title: "Kafka Instances"
         default: true
-      - id: service-accounts
-        title: "Service Accounts"
       - id: resources
         title: "Resources"
       - id: streams-docs
         title: Documentation
         navigate: https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/
 
+service-registry:
+  title: "Service Registry"
+  frontend:
+    sub_apps:
+      - id: sr
```

Comment:
```suggestion
      - id: service-registry
```

I think id should be `service-registry` instead 
@pmuir Please correct me if I'm wrong.

Reply:
Not clear tbh. Doing what @ajaypratap003 suggests will result in `application-services/service-registry/service-registry which is almost certainly wrong.

Reply:
This is a good point though @srijitamukherjee - I'm not sure that you put the nested routing properly into app-services -ui

Reply:
> This is a good point though @srijitamukherjee - I'm not sure that you put the nested routing properly into app-services -ui

@pmuir I can't see nested route in app-services-ui for service registry. We need to fix nested route based on what we want to keep id for SR. Can we have id  `registries` similar to kafkas

Reply:
I spoke to Tiffany - let's go with instances as the id